apps/src/componentLibrary/common/styles/colors.css

```
@@ -1,17 +1,22 @@
 @import "primitiveColors.css";
 
-/* This file consists of Semantic and Utility colors,
- if you need DSCO colors, you need to import and use this file.
+/* This file consists of Semantic colors,
+ if you need DSCO colors with support of multiple themes, you need to import and use this file.
  */
 
-/* Colors are taken from DISCO Variables New Figma File - Semantic colors and Utility colors variables collections
+/* Colors are taken from DISCO Variables New Figma File - Semantic colors colors variables collection
```

Should this be `DSCO`?
```suggestion
/* Colors are taken from DSCO Variables New Figma File - Semantic colors colors variables collection
```

---

Would it be worth linking to the Figma file here?

---

> Should this be DSCO?
Yep, thank you! Might had some mac t9 magic here

> Would it be worth linking to the Figma file here?

I asked design team if we want our Figma to be publicly available and answer was that we, at least for now, keep our working figma files private. So I think it's better not to include Figma file link for now

---

Cool, that makes sense!